<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CyberBox</class>
 <widget class="QMainWindow" name="CyberBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyberBox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelAttacker">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSwitch">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSN1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSN2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSN3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIT1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIT2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIT3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
